### DISCUSSION TITLE
Is it possible to use emojis in `Sandbox`?

### DISCUSSION THREAD

**Discussion by veniamin-ilmer** [hecrj/iced#1990] [Q&A] [answered]
I built small toy gui app, which has been using `Sandbox` thus far.

I would like to add a few emojis to it. The default font in my OS doesn't seem to support emojis.

Taking a look at examples ([color picker](https://github.com/iced-rs/iced_aw/blob/main/examples/color_picker/src/main.rs#L48) and [todos](https://github.com/iced-rs/iced/blob/master/examples/todos/src/main.rs#L67)), I see we have to load the font with `iced::font::load()`.

I attempted to run `font::load(include_bytes!("../fonts/icons.ttf").as_slice())` inside of my Sandbox `new` method, but this did not load the font.

I see `font::load` returns a `Command` object. Am I supposed to run this somehow?

`Application::new` can return `Command`, but `Sandbox` cannot, hence I am troubled, not understanding what I should do with this `Command` object.

**Comment by hecrj** [comment, marked as answer]
Unless the font you want to use isn't one of your system fonts, you should not need to load it. `iced` supports font fallback now.

However, you need to opt-in to the `Advanced` shaping strategy on any `Text` that needs it:

https://github.com/iced-rs/iced/blob/50ce65b3b7ad10a8537b751b3890d9dcfaecf846/examples/todos/src/main.rs#L368
